src/main/java/Interactors/SavePackager.java

```diff
@@ -0,0 +1,52 @@
+package Interactors;
+
+import Entities.Player;
+import Entities.Property;
+import TreeHandlers.GeneralGameLogic;
+import TreeHandlers.MainTreeNodeLogic.MainTreeNodeLogic;
+
+/***
+ * This class turns the objects of the game needed for data persistence into their needed forms for
+ * delivery to the SaveAccess methods.
+ */
+public class SavePackager {
+
+    public int[] getStates(){
+        int[] saveStates = new int[6];
+        int[] gameStates = MainTreeNodeLogic.getStates();
+        saveStates[0] = GeneralGameLogic.getBoard().getPlayers().size();
+        saveStates[1] = gameStates[4];
+        saveStates[2] = gameStates[2];
+        saveStates[3] = gameStates[3];
+        saveStates[4] = GeneralGameLogic.getCurrentPlayerIndex();
+        saveStates[5] = gameStates[1];
+
+        return saveStates;
+    }
+    public String[][] getPlayerPropertyData(){
```

Comment:
I think it would be a good idea to add Java Doc here in case we need to review this code in the future we have a general idea of what the logic is doing.

Reply:
Yep will add it now